tests/integration/maze_transformer/generation/test_plot_loss.py

```diff
@@ -0,0 +1,39 @@
+import os
+import tempfile
+from pathlib import Path
+
+import pytest
+
+from maze_transformer.evaluation.plot_loss import plot_loss
+from scripts.create_dataset import create_dataset
+from scripts.train_model import train_model
+
+
+@pytest.fixture()
+def temp_dir() -> Path:
+    data_dir = tempfile.TemporaryDirectory()
+    yield Path(data_dir.name)
+    data_dir.cleanup()
```

Comment:
This fixture is used in a few different files now.

We could try moving this fixture to a new module `tests.integration.fixtures` and importing it.

Reply:
May require a bit of research to figure out the optimal way to do this. 

More info here: https://gist.github.com/peterhurford/09f7dcda0ab04b95c026c60fa49c2a68


We could move this into a separate ticket, but that seems to me like the kind of the ticket which would never get prioritised.

Reply:
Great call, did the first step and added a comment for the second